components/ui-layout-view/src/UiLayoutViewWithRef.tsx

```diff
@@ -0,0 +1,14 @@
+import { forwardRef } from 'react';
+
+import { UiLayoutView } from './UiLayoutView';
+import { Props } from './UiLayoutView.types';
+
+export const UiLayoutViewWithRef = forwardRef<HTMLDivElement, Props>(
+  ({ children, ...props }, ref) => (
+    <UiLayoutView {...props} forwardedRef={ref}>
+      {children}
+    </UiLayoutView>
+  ),
+);
```

Comment:
Why we don't support `forwardRef` on the main component itself, instead of additional wrapper?

Reply:
To avoid extra HOC `forwardedRef`, where it isn't needed

Reply:
But we have the `forwardedRef` property in the `UiLayoutview`'s props, including typings in the same time. I vote for less entities! :D